src/components/@molecules/NFTTemplate/NFTTemplate.tsx

```diff
@@ -70,7 +71,7 @@ const NFTTemplate = ({ name, backgroundImage, isNormalised }: Props) => {
     if (isSubdomain && !name.includes('...')) {
       subdomain = `${labels.slice(0, labels.length - 2).join('.')}.`
       domain = labels.slice(-2).join('.')
-      if (getSegmentLength(subdomain) > MAX_CHAR) {
+      if (getSegmentLength?.(subdomain) > MAX_CHAR) {
```

Comment:
The "?." is unnecessary since getSegmentLength will always return a value. I added this when I was doing a quick test and had a different structure.